### PR TITLE
Added missing } and commented out DetailView.swift in MapScreen

### PR DESCRIPTION
**Pull request by anwar-ulhaq**
Changes to fix main

### CODE REVIEW

File: Voice Note/Views/MapScreen.swift [hunk 1]
```diff
@@ -19,15 +19,7 @@ struct MapScreen: View {
...
-                    DetailView(voiceNote: VoiceNote(
-                        noteId: UUID(),
-                        noteTitle: "Note - 1",
-                        noteText: "This place is good to come back in the summer. There are a lot of mushrooms and berries to pick up. Take good camera lens with me also for a good lanscape shot",
-                        noteDuration: TimeDuration(size: 3765),
-                        noteCreatedAt: Date.init(),
-                        noteTakenNear: "Ruoholahti",
-                        voiceNoteLocation: CLLocation(latitude: 24.33, longitude: 33.56)
-                    ))
+                    //DetailView(voiceNote)
```
Comment (anwar-ulhaq) [inline Voice Note/Views/MapScreen.swift:22]: after mains get fixed, i will uncomment

File: Voice Note/Views/DetailView.swift [hunk 2]
```diff
@@ -72,7 +72,7 @@ struct DetailBtn: View {
         }.padding(.horizontal,20).disabled(true)
     }
 }
-struct DetailView_Previews: PreviewProvider {
+/*struct DetailView_Previews: PreviewProvider {
```
Comment (anwar-ulhaq) [inline Voice Note/Views/DetailView.swift:75]: Uncomment after main gets fixed

File: Voice Note/Views/RecordingListView.swift [hunk 3]
```diff
@@ -34,6 +34,7 @@ struct RecordingListView: View {
                     ToolbarItem {
                         Button(action: addItem) {
                             Label("Add Item", systemImage: "plus")
+                        }
```
Comment (anwar-ulhaq) [inline Voice Note/Views/RecordingListView.swift:37]: Missed this closing brace while resolving conflict

**Review by giaongo — APPROVED** [review]
Approve the changes

**Pull request merged**